euclid.json

```diff
@@ -1,15 +1,14 @@
 {
-  "github_token": "",
   "version": "0.15.0",
-  "tessellation_version": "3.3.2",
+  "tessellation_version": "3.4.0-rc.12",
```

Comment:
should we use the latest tessellation of 3.4.14?
https://github.com/Constellation-Labs/tessellation/releases

Reply:
I think you're right, using `v3.4.0-rc.13` is most appropriate so that metagraphs are developed against a stable version of the network.